src/sentry/notifications/notifications/rules.py

```diff
@@ -109,6 +114,7 @@ def get_context(self) -> MutableMapping[str, Any]:
             "notification_reason": notification_reason,
             "has_alert_integration": has_alert_integration(self.project),
             "issue_type": GROUP_TYPE_TO_TEXT.get(self.group.issue_type, "Issue"),
+            "subtitle": self.event.title,
```

Comment:
Hmm, will this end up resulting in error alerts having a subtitle? Should we only add this when it's a generic email?

Reply:
Missed the if/else statement in the template, ignore this